styles/styles.css

```
@@ -132,46 +152,43 @@ li {
 
 .button {
     font-family: inherit;
-    display: flex;
-    flex-direction: row;
-    justify-content: center;
-    align-items: center;
     padding: 8px 50px;
-    gap: 10px;
     background-color: #82B3D3;
     color:#FFFFFF;
     font-size: 20px;
 }
 
-main {
-    font-family: inherit;
+.main-index {
     background-color: #FFFFFF;
 }
 
 .welcome {
     background-image: url(../images/index-background.jpg);
-    width: 1200px;
     background-repeat: repeat;
     background-size: cover;
-    background-position: 0%;
+    background-position: top;
```

```suggestion
    background-position: top center;
```
Все таки будет лучше, если будет по верху и по центру, а не по верху и левому краю.